contracts/ClaimsManager.sol

```diff
@@ -708,17 +702,25 @@ contract ClaimsManager is
         return accumulatedQuotaUsage - accumulatedQuotaUsageThen;
     }
 
-    function isManagerOrMediator(address account)
+    function isMediatorOrAdmin(address account)
         public
         view
         override
         returns (bool)
     {
         return
-            manager == account ||
             IAccessControlRegistry(accessControlRegistry).hasRole(
                 mediatorRole,
                 account
+            ) || isAdmin(account);
+    }
+
+    function isAdmin(address account) private view returns (bool) {
+        return
+            manager == account ||
+            IAccessControlRegistry(accessControlRegistry).hasRole(
+                adminRole,
+                account
```

Comment:
Previously `manager` was a superadmin that could only not downgrade policies (and [not resolve disputes on behalf of the arbitrator](https://github.com/api3dao/claims-manager/blob/2640e80c18db0a1388e5d72f0a1a8516a3d60e88/contracts/ClaimsManager.sol#L606) but that's mainly because doing so would put the arbitrator in a weird state). The admin role could do a lot of the highly privileged things (such as updating the oracle) but couldn't do things like creating policies, which bugged me while implementing the tests. So I did this for consistency.
I can imagine a case where the manager is so cumbersome that it really doesn't want to do anything (e.g. it's the DAO) and needs to be able to delegate all of its privileges to a contract that can exercise them based on certain rules, which is why I did it this way. I don't see a point in using the admin role if manager is a multisig, doing things as the manager should be easy enough.